rules/windows/lateral_movement_rdp_enabled_registry.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/11/25"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/25"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies registry write modification to enable RDP access. This could be indicative of adversary lateral movement
+preparation.
```

Comment:
```suggestion
Identifies registry write modifications to enable Remote Desktop Protocol (RDP) access. This could be indicative of adversary lateral movement
preparation.
```

Reply:
re-lint